src/index.njk

```diff
@@ -5,11 +5,11 @@ layout: base.njk
 {% from 'macros/cards.njk' import feedCard %}
 {% include 'partials/banner.njk' %}
 <main>
-  <div class="mx-auto w-full bg-gray-300 items-center">
-    <div class="py-10 px-5 md:px-20 bg-gray-300 dark:bg-gray-800">
+  <div class="mx-auto w-full items-center">
+    <div class="py-10 px-5 md:px-20 bg-ivory dark:bg-darkGray">
```

Comment:
Please apply a color class different from that of card sections for light mode, to maintain visual distinction between page background and content cards when the site UI is in light mode.